CONTRIBUTING.md

```diff
@@ -1,8 +1,8 @@
 # Contributing
 
-Demonstrating Engineering Fundamentals is core to what we do in ISE and is one of the primary values we bring to our customers, helping them to level up while collaborating on their business scenarios.
+Demonstrating Engineering Excellence is core to what we do at Neudesic and is one of the primary values we bring to our customers, helping them to level up while collaborating on their business scenarios.
 
-The purpose of this repo is to provide guidance to ISE software engineers and data scientists (together forming a Dev Crew) with regards to engineering fundamentals. It describes recommended practices based on continuous project learnings in different areas. It can be used as a tool at the beginning of an engagement that can be shared with customers to help set the project up for success. This repo is *not* intended as a code repo, instead it provides guidance and links to appropriate sample code repos that represent good examples.
+The purpose of this repo is to provide guidance to software engineers, infrastructure engineers, data engineers, and data scientists (together forming a Dev Crew) with regards to engineering fundamentals. It describes recommended practices based on continuous project learnings in different areas. It can be used as a tool at the beginning of an engagement that can be shared with customers to help set the project up for success. This repo is *not* intended as a code repo, instead it provides guidance and links to appropriate sample code repos that represent good examples.
```

Comment:
First time I'm seeing reference to the term 'dev crew'.  Is this a new definition\term (I get the concept) in the firm?  If we don't have this clearly defined in other spaces/conversations, is there a more Neudesic-terminology-appropriate phrase here?  Dev Squad?  Solution Team?  Other?

Reply:
It's taken straight from MSFT and we can certainly change it. Any suggestions? 

Added as #19